taskcluster/balrog_taskgraph/transforms/docker_push.py

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+Set environment variables for the skopeo push-image command
+"""
+
+
+import os
+
+from taskgraph.transforms.base import TransformSequence
+
+transforms = TransformSequence()
+
+
+@transforms.add
+def set_push_environment(config, jobs):
+    """Set the environment variables for the push to docker hub task."""
+    for job in jobs:
+        version_file = os.path.join(config.graph_config.vcs_root, "version.txt")
+        version = open(version_file).read().strip()
+
+        env = job["worker"].setdefault("env", {})
+        env.update(
+            {
+                "VCS_HEAD_REPOSITORY": config.params["head_repository"],
+                "VCS_HEAD_REV": config.params["head_rev"],
+                # TODO: Figure out if we still need to add the version
```

Comment:
Want to address the TODO now?

Reply:
There's two places we have the version number for a release:
- version.txt; taskgraph also reads that and fills in the "version" param (https://github.com/taskcluster/taskgraph/blob/591768b3fd5b8aa50e003e43d29cc849cd880388/src/taskgraph/parameters.py#L121)
- the tag name, which also ends up as the "head_tag" param

I figure we should put a check in the github-release decision task to abort if the two aren't the same.  I'll remove this TODO, though, because as far as I can tell we do need to pass the version to the push-image task which uses it to generate `/app/version.json` in the image.